omod/src/main/java/org/openmrs/module/webservices/rest/resource/StockOperationResource.java

```diff
@@ -188,6 +189,11 @@ public void setStatus(StockOperation operation, StockOperationStatus status) {
 		}
 	}
 
+	@PropertySetter("cancelReason")
```

Comment:
I don't think that this property setter is needed. Do you run into an error without it?